src/Impostor.Server/Net/Inner/Objects/InnerGameData.cs

```diff
@@ -172,12 +172,12 @@ private void SetTasks(byte playerId, ReadOnlyMemory<byte> taskTypeIds)
                 return;
             }
 
-            player.Tasks = new List<TaskInfo>(taskTypeIds.Length);
+            player.Tasks = new List<ITaskInfo>(taskTypeIds.Length);
 
             for (var i = 0; i < taskTypeIds.Length; i++)
             {
-                player.Tasks.Add(new TaskInfo());
-                player.Tasks[i].Id = (uint)i;
+                var task = new TaskInfo { Id = (uint)i };
+                player.Tasks.Add(task);
```

Comment:
does not close  #140 
still missing TaskInfo data from `taskTypeIds`

Reply:
What do you mean?

Reply:
player.Tasks is empty need deserialize `taskTypeIds`

Reply:
Resolved in 2667387